### DISCUSSION TITLE
app status

### DISCUSSION THREAD

**Discussion by WorkingSilence-L** [amzn/selling-partner-api-models#3674] [General]
I have two questions and I look forward to receiving a response：
1、If the APP is in review   or APP  is approved and being prepared for publishing，will it affect authorized sales partners in the draft status of the application to obtain data
2、Do sales partner app that have already been authorized in draft status need to be re authorized after they are Published

**Comment by rugved1991** [comment]
Hi @WorkingSilence-L, 

1. No it wont affect authorized sellers to call APIs.
2. The authorized selling partners dont need to be re-authorized after app publish. Unless there were any changes in the roles of the application after initial authorization.

Best,
Rugved
Solutions Architect, SP-API
